telegram/ext/utils/promise.py

```diff
@@ -83,6 +84,8 @@ def run(self) -> None:
 
         finally:
             self.done.set()
+            if self._done_callback:
```

Comment:
We should do some `try-except` here for completeness. IISC we can only reasonably `pass` on exceptions, but it's better to make sure that ony exceptions here don't break anything …